zipkin-ui/webpack.config.js

```diff
@@ -27,9 +27,9 @@ module.exports = {
         }]
     },
     output: {
-        path: __dirname + '/dist/static',
+        path: __dirname + '/build/resources/main/static/assets',
```

Comment:
Sure thing. I'll be putting down Zipkin soon for the day; I'd say let's wait for more input on https://github.com/openzipkin/zipkin-java/pull/98#issuecomment-197361662, that'll probably affect this as well.